.github/workflows/internal-action-test.yml

```diff
@@ -43,7 +45,8 @@ jobs:
           // artifact images contain only a single platform.
           if (process.env.ACTION_CHANGED === 'true' && '${{ inputs.artifact }}' === '') {
             os.push('ubuntu-24.04-arm');
-            os.push('macos-13');
+            os.push('macos-latest');
```

Comment:
I don't expect macos-latest (arm) to work, because the macos cpus on github are M1 cpus, that do not support virtualisation required for running docker. See https://docs.github.com/en/actions/reference/runners/github-hosted-runners
```suggestion
```